src/reducers/index.js

```diff
@@ -1,25 +1,28 @@
-import * as types from '../constants/ActionTypes';
-import omit from 'lodash/object/omit';
-import assign from 'lodash/object/assign';
-import mapValues from 'lodash/object/mapValues';
+import { combineReducers } from 'redux';
+import { CHANGE_LOOK } from '../actions';
 
 const initialState = {
   multipleOutFitLook: 1
 };
 
-export default function friends(state = initialState, action) {
+function look(state = initialState, action) {
```

Comment:
You'll need more than one reducer in app, so I won't export the one you wrote directly as default.